### ISSUE TITLE
Gnome Shell 3.32.0 (Ubuntu 19.04) 3 finger settings actually 4

### ISSUE THREAD

**Issue by joedborg** [mpiannucci/gnome-shell-extended-gestures#64]
Using GNOME Shell 3.32.0 on Ubuntu 19.04.

![Screenshot from 2019-03-21 10-22-23](https://user-images.githubusercontent.com/1426404/54746112-46cc2280-4bc3-11e9-9e54-63faaa1a4fff.png)

These are my settings.  Cycle applications (3 finger horizontal) works fine, but cycle workspaces (3 finger vertical), actually requires 4 fingers.  Not sure why this is?  Is there any debugging I can do?

**Comment by dogmax** [comment]
This happens on 18.10 as well.

**Comment by mpiannucci** [comment]
Hmm thats interesting. Does the four finger gesture obey the action you defined in the GUI? if not I am thinking Gnome has hijacked it and may require a workaround

**Comment by joedborg** [comment]
@mpiannucci thanks for the reply.  It doesn't let me tick those 4 finger gestures to enable them.

**Comment by mpiannucci** [comment]
yeah thats by default. Can you change the gesture for three finger vertical and see if it works? 

By default Gnome handles the four finger vertical gesture and I have not been able to override it.

**Comment by joedborg** [comment]
Ahh, I did not know 4 finger was a gnome built in for the workspace swap.  Is this a new thing?

Yes, you're right, if I change the behavior, 3 finger works.  It's just swallowing the workspace switching.
